source/includes/v2/_signing-data.md

```diff
@@ -0,0 +1,75 @@
+# V2 requests signing
+
+V2 API supports signed requests with asymmetric cryptography. Since every type of request is structurally different, the
+format of the data to be signed also varies. In order to sign a request, clientside has to put together a string 
+in the specific format and sign that string with the customer's private key, thereby generating RSA signature. 
+Every V2 API definition contains special-purpose optional field called "Signature", which has to be used to pass 
+the RSA signature to the serverside.
+
+The specific formats of the data to be signed are defined on the respective pages, describing an API endpoint. For clarity,
+each API endpoint page contains subsection called "Signing data format" with detailed description of the parameters 
+to be used for putting together textual data for signing.
+
+For convenience of use, links to all formats' descriptions are compiled below:
+
+- *[Withdrawal V2 execution signing data format](_brokerage-api-withdrawals.md#signing-data-format)*
+
+## Signing format guidelines
+
+While all requests serve various purposes and therefore consist of different parameters, they also follow several common
+patterns. The way some of these patterns are used across V2 APIs in relation to the signing functionality is described below:
+
+### General format description
+
+Signing data is a sequence of bytes, corresponding to UTF-8-encoded string in the specific format. The beginning and ending 
+of the signing data are denoted by square brackets: `[]`. Each parameter value is encased in single quotes: `''`.
```

Comment:
what if a value contains a single quite? how it should be escaped?

Reply:
`+` Implemented escaping in Guardian and added subsection here with elaboration how it works